### PR TITLE
Mutual TLS Sample

### PR DESCRIPTION
**Pull request by philsturgeon**
Fixes https://github.com/OAI/OpenAPI-Specification/issues/2475

### CODE REVIEW

**Comment by MikeRalphson** [comment]
The `type` value is `mutualTLS` not `tls`.

File: versions/3.1.0.md [hunk 1]
```diff
@@ -3253,6 +3253,20 @@ scheme: bearer
 bearerFormat: JWT
 ```
 
+###### MutualTLS Sample
+
+```json
+{
+  "type": "mutualTLS",
+  "description": "Cert must be signed by evilcorp.com CA"
```
Comment (darrelmiller) [inline versions/3.1.0.md:3261]:
```suggestion
  "description": "Cert must be signed by example.com CA"
```

File: versions/3.1.0.md [hunk 2]
```diff
@@ -3253,6 +3253,20 @@ scheme: bearer
...
+}
+```
+
+```yaml
+type: mutualTLS
+description: Cert must be signed by evilcorp.com CA
```
Comment (darrelmiller) [inline versions/3.1.0.md:3267]:
```suggestion
description: Cert must be signed by example.com CA
```

**Review by darrelmiller — CHANGES_REQUESTED** [review]
Hey Phil, please approve the new name.

**Comment by philsturgeon** [comment]
Done!

**Review by darrelmiller — APPROVED** [review]

**Comment by commonism** [comment]
this was merged into versions/3.1.0.md in the v3.1.1-dev branch
I' expect it to be in versions/3.1.1.md and 3.2.0.md

**Pull request merged**